modules/logging-loki-storage-aws.adoc

```diff
@@ -1,18 +1,21 @@
 // Module is included in the following assemblies:
-// logging/cluster-logging-loki.adoc
+// logging/log_storage/installing-log-storage.adoc
 //
 :_mod-docs-content-type: PROCEDURE
 [id="logging-loki-storage-aws_{context}"]
 = AWS storage
```

Comment:
Procedure titles should begin with a gerund, such as "Configuring AWS storage".

Reply:
Out of scope